libs/tracing/ppx/ppx_tracing.ml

```diff
@@ -83,22 +91,21 @@ let module_name_of_loc loc =
 
 (* To produce arguments like `~__FILE__`*)
 let make_label loc l =
-  ( Labelled l,
-    {
-      pexp_desc = Pexp_ident { txt = Lident l; loc };
-      pexp_loc_stack = [];
-      pexp_loc = loc;
-      pexp_attributes = [];
-    } )
-
-let make_traced_expr loc action_name var_pat e =
+  (Labelled l, Exp.mk (Pexp_ident { txt = Lident l; loc }) ~loc)
+
+let make_traced_expr ~level loc action_name var_pat e =
   Exp.apply
     (Exp.ident { txt = Lident "@@"; loc })
     [
       ( Nolabel,
         Exp.apply
           (Exp.ident { txt = Ldot (Lident "Tracing", "with_span"); loc })
           [
+            ( Labelled "level",
+              Exp.mk ~loc
+                (Pexp_construct
+                   ( { txt = Ldot (Lident "Tracing", level_to_str level); loc },
```

Comment:
If the level is just being used to eventually pass down and become a string here, is there a reason we can't just use `Tracing.level` instead, and have `Tracing.show_level` instead of `level_to_str`?

Reply:
I was convinced that there would be a circular dependency but it seems like it's actually fine